tests/test_aiohttp_responds_with_decorator.py

```diff
@@ -0,0 +1,203 @@
+import contextlib
+import io
+import pytest
+from aiohttp import web
+from schematics import Model
+from schematics.types import StringType, IntType
+from unittest import mock
+import json
+
+from apistrap.aiohttp import ErrorHandlerMiddleware
+from apistrap.errors import UnexpectedResponseError, InvalidResponseError
+from apistrap.types import FileResponse
+
+
+class OkResponse(Model):
+    string_field = StringType(required=True)
+
+
+class ErrorResponse(Model):
+    error_message = StringType(required=True)
+
+
+class WeirdResponse(Model):
+    string_field = StringType(required=True)
+
+
+class RequestModel(Model):
+    string_field: str = StringType(required=True)
+    int_field: int = IntType(required=True)
+
+
+@pytest.fixture()
+def app_with_responds_with(aiohttp_apistrap, tmpdir):
+    app = web.Application()
+    routes = web.RouteTableDef()
+
+    @routes.get('/')
+    @aiohttp_apistrap.responds_with(OkResponse)
+    @aiohttp_apistrap.responds_with(ErrorResponse, code=400)
+    async def view(request):
+        return OkResponse(dict(string_field='Hello World'))
+
+    @routes.get('/error')
+    @aiohttp_apistrap.responds_with(OkResponse)
+    @aiohttp_apistrap.responds_with(ErrorResponse, code=400)
+    async def error(request):
+        return ErrorResponse(dict(error_message='Error'))
+
+    @routes.get('/weird')
+    @aiohttp_apistrap.responds_with(OkResponse)
+    @aiohttp_apistrap.responds_with(ErrorResponse, code=400)
+    async def weird(request):
+        return WeirdResponse(dict(string_field='Hello Weird World'))
+
+    @routes.get('/invalid')
+    @aiohttp_apistrap.responds_with(OkResponse)
+    @aiohttp_apistrap.responds_with(ErrorResponse, code=400)
+    async def invalid(request):
+        return OkResponse(dict())
+
+    @routes.get("/file")
+    @aiohttp_apistrap.responds_with(FileResponse)
+    async def get_file(request):
+        message = 'hello'
+        return FileResponse(filename_or_fp=io.BytesIO(message.encode('UTF-8')),
+                            as_attachment=True,
+                            attachment_filename='hello.txt')
+
+    @routes.get("/file_by_path")
+    @aiohttp_apistrap.responds_with(FileResponse)
+    async def get_file_by_path(request):
+        path = tmpdir.join("file.txt")
+        path.write(b'hello')
+
+        return FileResponse(filename_or_fp=str(path),
+                            as_attachment=True,
+                            attachment_filename='hello.txt')
+
+    @routes.get('/file_without_name')
+    @aiohttp_apistrap.responds_with(FileResponse)
+    async def get_file_without_name(request):
+        message = 'hello'
+        return FileResponse(filename_or_fp=io.BytesIO(message.encode('UTF-8')),
+                            as_attachment=True)
+
+    @routes.get('/file_with_mimetype')
+    @aiohttp_apistrap.responds_with(FileResponse)
+    async def get_file_with_mimetype(request):
+        message = '<a href = "www.hello.com"</a>'
```

Comment:
```suggestion
        message = '<a href="www.hello.com"></a>'
```

Reply:
...Could you just make a commit where you make all these changes? Having four commits like this is stupid.